packages/ra-ui-materialui/src/input/ResettableTextField.tsx

```diff
@@ -26,6 +26,7 @@ export const ResettableTextField = forwardRef(
             variant,
             margin,
             className,
+            size = 'small',
```

Comment:
I think this doesn't fix the issue, as the underlying MUI TextField will still receive `size="small"`, while we want it to receive nothing so it ca use the theme override.

Reply:
You are right.
I'll change it